packages/cdktf/lib/testing/matchers.ts

```diff
@@ -29,8 +31,13 @@ export function returnMatcherToJest(
   };
 }
 
-// All expected properties are matched and considered equal if
-// There can be more properties in the received object than in the expected object while still returning true
+/**
+ * Compares expected and received. All expected properties are matched and considered equal if
+ * there can be more properties in the received object than in the expected object while still returning true.
```

Comment:
Ah, okay but the sentence does not sound right. "…considered equal if there can be more…" 😅

```suggestion
 * Compares expected and received. All expected properties are matched and considered equal even if
 * there are more properties in the received object than in the expected object in which case it will still return true.
```